db/models/models.js

```diff
@@ -0,0 +1,11 @@
+const db =require('../connection');
+
+exports.fetchTopics=()=>{
+    return db.query('SELECT * from topics;')
+    .then((response)=>{
+        return response.rows;
+    }).catch((err)=>{
+        next(err)
+    });
+
+}
```

Comment:
Try and use catch blocks in your controller rather than your model as the controller will be at the end of the Promise chain.